src/tests/ftest/performance/ior_easy.py

```diff
@@ -0,0 +1,127 @@
+#!/usr/bin/env python3
+"""
+  (C) Copyright 2018-2021 Intel Corporation.
+
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+"""
+from performance_test_base import PerformanceTestBase
+
+class IorEasy(PerformanceTestBase):
+    # pylint: disable=too-many-ancestors
+    # pylint: disable=too-few-public-methods
+    """Test class Description: Run IOR Easy
+
+    Use Cases:
+            Create a pool, container, and run IOR Easy.
+
+    :avocado: recursive
+    :avocado: tags=performance,performance_ior,performance_ior_easy
```

Comment:
These tags are applied to all `test_*` methods for this class.

Reply:
The main reason is because there are so many variants in this file, and there will probably be more in the future. The second reason was to serve as an example, for a second opinion. Do you think this is too confusing?

Reply:
That's fine. Don't think it's confusing. Makes sense if you you want to tackle the situation of many variants and more to come in future.

Reply:
We should probably discuss this option.  While it is more efficient, I could see a case where someone just searches for `test_performance_ior_easy_mpiio_sx`, for example, and misses that fact that there are other tags defined in the class docstring as it would most likely be off their screen.  Depending upon what tags are specified in the class docstring, it could be confusing to folks looking for a `Test-tag:` or `Features:` entry who are not familiar with tags.  Not enough of an issue to prevent landing though.

Reply:
I see your point - will discuss in WG, etc. I do have another PR queued where I can change this if there are no other required changes for this PR.